hackathon_site/dashboard/frontend/src/api/api.ts

```diff
@@ -86,3 +86,8 @@ export const post = <T>(uri: string, data?: any) => {
     uri = cleanURI(uri);
     return axios.post(`${SERVER_URL}/${uri}`, data, makeConfig());
 };
+
+export const patch = <T>(uri: string, data?: any) => {
```

Comment:
This was missing from post function but can you make the function return type `Promise<AxiosResponse<T>>`